File: c/1014_consumption.c
/*  
    @autor: Guiventura6;
    @data: 20/01/2022;
    @Problema: Consumption;
*/
#include <stdio.h>

int main(){

    int totalDistance;
    double spentFuelTotal;

    scanf("%d", &totalDistance);
    scanf("%lf", &spentFuelTotal);

    double averageConsumption = totalDistance / spentFuelTotal;

    printf("%.3lf km/l\n", averageConsumption);
    
    return 0;
}

File: python/1014_consumption.py
'''
    Autor: Guiventura6
    Data: 02/03/2022
    Modified: (27/06/2023,)
    Problema: Consumption
    Resume: Program that reads the total distance traveled(km) and the spent
    fuel total(L) and calculate a car's average consumption
'''
total_distance = int(input())
total_fuel = float(input())

average_consumption = total_distance / total_fuel

print('{:.3f} km/l'.format(average_consumption))
